src/apis/drive/samples/node/quickstart.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018, Google, LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START main_body]
+const {drive, auth} = require('@google/drive');
+const express = require('express');
+const opn = require('opn');
+const path = require('path');
+const fs = require('fs');
+
+const keyfile = path.join(__dirname, '..', 'oauth2.keys.json');
```

Comment:
Thinking out loud - should we rewrite these to use whatever is in `GOOGLE_APPLICATION_CREDENTIALS` for finding the path to the creds file?  That would be consistent with what we do for service account credentials 🤷‍♂️

Reply:
Looks like a good idea, even though this JSON file is different compared to the service account key.

Reply:
...but since I'm trying not to change samples too much, I guess this change will be out of scope :)